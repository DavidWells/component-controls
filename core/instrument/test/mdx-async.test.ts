import { loadStoriesTests } from './loadTestFiles';

describe('mdx-async', () => {
  loadStoriesTests(undefined, ['mdx', 'async']);
});
