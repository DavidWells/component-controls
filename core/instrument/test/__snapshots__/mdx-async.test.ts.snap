// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mdx-async arrow-function.mdx 1`] = `
Object {
  "components": Object {
    "fbc7c3606fe52918c0716b2079a81097": Object {
      "from": "../components/BaseButton",
      "importedName": "default",
      "name": "Button",
    },
  },
  "doc": Object {
    "component": "Button",
    "componentsLookup": Object {
      "Button": "fbc7c3606fe52918c0716b2079a81097",
    },
    "date": 2020-10-15T00:19:23.169Z,
    "dateModified": 2020-10-15T00:19:23.172Z,
    "package": "74d5ae881b6029d05c88093109de57a1",
    "title": "Storybook/MDX",
  },
  "packages": Object {
    "74d5ae881b6029d05c88093109de57a1": Object {
      "dependencies": Object {
        "@babel/generator": "^7.9.4",
        "@babel/parser": "^7.9.4",
        "@babel/traverse": "^7.9.0",
        "@component-controls/core": "^1.32.5",
        "@hutson/parse-repository-url": "^5.0.0",
        "@mdx-js/loader": "^1.5.5",
        "@mdx-js/react": "^1.6.5",
        "camelcase": "^6.0.0",
        "deepmerge": "^4.2.2",
        "find-cache-dir": "^3.3.1",
        "gray-matter": "^4.0.2",
        "hosted-git-info": "^3.0.4",
        "js-string-escape": "^1.0.1",
        "prettier": "^1.19.1",
        "read-package-json": "^2.1.1",
        "remark-emoji": "^2.1.0",
        "remark-images": "^2.0.0",
        "resolve": "^1.15.1",
        "typescript": "^3.8.3",
      },
      "devDependencies": Object {
        "@babel/types": "^7.9.0",
        "@component-controls/ts-markdown-docs": "^1.21.0",
        "@rollup/plugin-node-resolve": "^7.1.1",
        "@types/find-cache-dir": "^3.2.0",
        "@types/hosted-git-info": "^3.0.0",
        "@types/jest": "^26.0.10",
        "cross-env": "^5.2.1",
        "eslint": "^6.5.1",
        "jest": "^26.4.2",
      },
      "fileHash": "74d5ae881b6029d05c88093109de57a1",
      "name": "@component-controls/instrument",
      "peerDependencies": undefined,
      "repository": Object {
        "browse": "https://github.com/ccontrols/component-controls/tree/master/core/instrument/test/fixtures/mdx/async/arrow-function.mdx",
        "docs": "https://github.com/ccontrols/component-controls/tree/master#readme",
        "issues": "https://github.com/ccontrols/component-controls/issues",
      },
      "version": "1.33.1",
    },
  },
  "stories": Object {
    "storyWithProps": Object {
      "arguments": Array [
        Object {
          "loc": Object {
            "end": Object {
              "column": 11,
              "line": 0,
            },
            "start": Object {
              "column": 6,
              "line": 0,
            },
          },
          "name": "props",
          "value": "props",
        },
      ],
      "id": "storyWithProps",
      "loc": Object {
        "end": Object {
          "column": 63,
          "line": 12,
        },
        "start": Object {
          "column": 38,
          "line": 12,
        },
      },
      "name": "story-with-props",
      "source": "async props => <Button />",
    },
  },
  "transformed": "
    
    import { Story, Meta } from '@component-controls/blocks';
import Button from '../components/BaseButton';


<Meta title=\\"Storybook/MDX\\" component={Button} />

<Story name=\\"story-with-props\\">
  {async (props) => <Button />}
</Story>

",
}
`;
